sec_certs/configuration.py

```diff
@@ -2,16 +2,25 @@
 from typing import Union, Any
 from pathlib import Path
 from importlib_resources import files
+import jsonschema
+import json
+import os
 
 import sec_certs
 
 
 class Configuration(object):
-    # TODO: Should raise ValueError on unvalidated config
     def load(self, filepath: Union[str, Path]):
         with Path(filepath).open('r') as file:
             state = yaml.load(file, Loader=yaml.FullLoader)
 
+        script_dir = os.path.dirname(os.path.realpath(__file__))
+
+        with open(Path(script_dir) / 'settings-schema.json', 'r') as file:
```

Comment:
You need to reference `settings-schema.json` at [setup.py](https://github.com/crocs-muni/sec-certs/blob/2863d099992cb4c275e543615617e0c589193315/setup.py#L37) under package data. Otherwise, `json` won't be included after package installation.